doc/MIGRATION-v5.md

```diff
@@ -0,0 +1,6 @@
+# Migration from 4.x to 5.x
+
+## kube-validation-schema.json files renamed to validation-schema.json
```

Comment:
Why this migration guide doesn't provide information about `Doneable` being removed and change in `edit()` signatures? Or does this PR only document it from the model perspective?